posts/taking-learning-outside-the-classroom-into-the-cloud.md

```diff
@@ -0,0 +1,54 @@
+
+As the application progressed, I had the opportunity to design the user experience for authoring a new assessment from scratch, which involved creating hierarchies of categories, questions, and answers. I ended up developing both the front and back ends of the component for creating questions and answers, so I was actually able to watch my ideas evolve from conception to implementation into production. It was immensely rewarding to not only design a user experience for the first time, but then enable its full development, knowing the final product would serve the company in a meaningful way.
+
+
+<h1>Developing Full-Time with a Client, Summer 2020 – Spring 2021 </h1>
+
+When I first pictured myself working full-time in fraud prevention for a banking client, I imagined myself pouring over database records, constructing complex queries to catch fraud, and I was intimidated, to say the least. While there are indeed analysts in the trenches running queries to discover new patterns and sources of fraud, I was naive to my role as an engineer in the process of stopping fraud. The team I worked with for the past 9 months in fact maintains the engine that runs analyst-authored rules to detect fraudulent transactions in real time. My team also developed and maintains a platform for enabling A/B testing of such rules against near-real data before sending the rules off to production, eliminating the risks of releasing malauthored rules and enabling `intent = execution` validation. While the team and organization as a whole still devote a considerable amount of developer time to ‘keeping the lights on,’ much of my work over the past 9 months contributed to a larger effort to automate various processes and ultimately unlock more developer bandwidth to continue iterating on and innovating the next version of the platform.
+
+My first engagement with this team involved upgrading our rule testing platform to a higher tier of resiliency, by making the infrastructure “active/active.” This being my first assignment, I was of course intimidated but moreover excited to put my abilities to learn on-the-fly to test. I paired with a teammate who joined at the same time I did, and together we learned about Jenkins pipelines, AWS Lambda/ELB/EC2/Route53 and their interactions with one another, as well as how to deploy them all using the company's custom deployment processes. These technologies were just the tip of the iceberg-sized tech stack that I worked with over the next 9 months: among others, I got hands-on experience with Docker, Kubernetes, and Maven (maybe even a little too much experience with Maven dependency management... if you know, you know).
```

Comment:
Despite being intimidated with my first assignment, I was also excited to put my abilities to learn on-the-fly to the test.